gcp/basic-auth-ingress/base/kustomization.yaml

```diff
@@ -15,12 +15,12 @@ namespace: kubeflow
 commonLabels:
   kustomize.component: basic-auth-ingress
 images:
-  - name: gcr.io/kubeflow-images-public/ingress-setup
-    newName: gcr.io/kubeflow-images-public/ingress-setup
-    newTag: latest
-  - name: gcr.io/cloud-solutions-group/esp-sample-app
-    newName: gcr.io/cloud-solutions-group/esp-sample-app
-    newTag: 1.0.0
+- name: gcr.io/kubeflow-images-public/ingress-setup
+  newName: gcr.io/kubeflow-images-public/ingress-setup
+  newTag: latest
```

Comment:
Could you change this to
digest: sha256:91db78e53a3b87cf64829c096b25d6b4ab0770b28fec0f7641a0e1707d12c83c

So we pin to a specific image.